src/app/COMPONENTS/task/task.component.ts

```diff
@@ -27,16 +28,15 @@ import { MaterialModule } from '../../material.module';
 
 export class TaskComponent {
 
-  availableTasks$ = this.taskService.getAllTasks()
+  availableTasks$ = this.taskService.getAllTasks().pipe(take(1))
```

Comment:
you don't have to use the take operator here since you are using the async pipe for this variable, it will automatically unsubscribe